lib/src/ledger/ledger_ble_manager.dart

```diff
@@ -117,10 +108,6 @@ class LedgerBleConnectionManager extends ConnectionManager {
         : BleConnectionState.disconnected;
     final controller = await _getOrCreateConnectionStateController(deviceId);
     controller.add(state);
-
-    if (!isConnected) {
-      await disconnect(deviceId);
-    }
```

Comment:
I don't fully remember what's the deal with this, but I believe it's also to clear the connection in case the device is disconnected because of other reasons.

Reply:
I get the the PlatformException here too. From my testing this gets called when the device disconnects and after that there is not much to clean up. I clean up everything with a listener on `deviceStateChanges`